helpers/util.sh

```diff
@@ -325,17 +329,18 @@ build() {
     # The release number of these packages change each time the .spec is queried (contains
     # timestamp), so mb2 would fail to preserve the latest (just built) packages. Use
     # --package-timeline as a workaround.
+    set -x
     local PACKAGE_TIMELINE=
     [[ $PKG == droid-hal-* ]] || [[ $PKG == droid-configs ]] && PACKAGE_TIMELINE=--package-timeline
 
     for SPEC in "$@" ; do
         minfo "Building $SPEC"
         mb2 \
             -s $SPEC \
-            -t $VENDOR-$DEVICE-$PORT_ARCH \
+            -t $HABUILD_TARGET \
             $PACKAGE_TIMELINE $NO_AUTO_VERSION \
             --output-dir "$LOCAL_REPO" \
-            build >>$LOG 2>&1|| die "building of package failed"
+            build --no-check >>$LOG 2>&1|| die "building of package failed"
```

Comment:
I've built packages for `i3113`, then ran `export DEVICE=i4113` and mb2 is still checking places of the previous device:
```
PlatformSDK $ANDROID_ROOT $ rpm/dhd/helpers/build_packages.sh --version
* Building rpm/droid-hal-version-i4113.spec 
Checking whether to refresh metadata for dir:$ANDROID_ROOT/hybris/droid-hal-version-i3113/.mb2/filtered-output-dir
```

Reply:
Hm for kumano this works fine for me: 
```
bidar@odin:~/dev/sailfish/hadk/kumano|21:59$ ./rpm/dhd/helpers/build_packages.sh --version
* Building rpm/droid-hal-version-j9210.spec 
Checking whether to refresh metadata for dir:/home/bidar/dev/sailfish/hadk/kumano/hybris/droid-hal-version-sony-kumano/.mb2/filtered-output-dir
```